### PR TITLE
Ci/deploy heroku

### PR DESCRIPTION
**Pull request by ramirobg94**

### CODE REVIEW

**Review by sorja — APPROVED** [review]

File: .github/workflows/test.js.yml [hunk 1]
```diff
@@ -0,0 +1,84 @@
+name: Run yarn test
+
+on:
+  push:
+    branches-ignore: 
+      - 'master'
+      - 'production'
```
Comment (minotogna) [inline .github/workflows/test.js.yml:7]: this must run on pull_request

**Review by minotogna — CHANGES_REQUESTED** [review]

**Review by sorja — APPROVED** [review]

File: .github/workflows/test.js.yml [hunk 2]
```diff
@@ -0,0 +1,83 @@
+name: Run yarn test
+
+on:
+  pull_request:
+    -branches:
+      - 'master'
```
Comment (minotogna) [inline .github/workflows/test.js.yml:6]: can this run on all pull requests? not only to master

Comment (ramirobg94) [inline .github/workflows/test.js.yml:6]:
```suggestion
```

File: .github/workflows/test.js.yml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+name: Run yarn test
+
+on:
+  pull_request:
+    -branches:
```
Comment (ramirobg94) [inline .github/workflows/test.js.yml:5]:
```suggestion
```

**Review by minotogna — APPROVED** [review]

**Pull request merged**